### ISSUE TITLE
Port to wiki.bedrock.dev?

### ISSUE THREAD

**Issue by SirLich** [skylinerw/guides#25]
Hi! I am the curator of https://wiki.bedrock.dev/

Would it be acceptable for me to provide a copy of this document there? Otherwise, I suppose I could hot-link it.

**Comment by skylinerw** [comment]
Sure, just be aware that a lot of it is outdated (especially the bedrock loot tables bit).

**Comment by SirLich** [comment]
Yeah I am aware its out of date. One of the reasons I wanted to maintain it myself!
